contracts/facets/Market/MarketFacet.sol

```diff
@@ -171,24 +169,25 @@ contract MarketFacet is IMarket {
             // // If a bid meets the criteria for an ask, automatically accept the bid.
             // // If no ask is set or the bid does not meet the requirements, ignore.
             if (
-                _bid._amount >= ms._tokenAsks[_tokenID]._askAmount
+                _bid._amount >= ms._tokenAsks[_tokenAddress][_owner][_tokenID]._askAmount
             ) {
+                ms._tokenAsks[_tokenAddress][_owner][_tokenID]._amount -= _bid._quantity;
```

Comment:
This should be decreasing `._quantity` not `._amount`, right?

Reply:
in ask struct, its basically maintain in amount variable. I mentioned in todays call. I will fix them after your merge this PR